### PR TITLE
new env update

### PR DESCRIPTION
**Pull request by akshaybatar**
ok chelga ye

### CODE REVIEW

**Review by akshaybatar — COMMENTED** [review]
ok

File: .env.example [hunk 1]
```diff
@@ -11,7 +11,7 @@ LOG_LEVEL=debug
 DB_CONNECTION=mysql
 DB_HOST=127.0.0.1
 DB_PORT=3306
-DB_DATABASE=codeupdatede
+DB_DATABASE=branchchanged
```
Comment (akshay8126) [inline .env.example:14]: ok

**Review by akshaybatar — COMMENTED** [review]
ok

**Comment by akshay8126** [comment]
kmm

**Pull request merged**